Swiftfin/Views/VideoPlayer/LiveTVPlayerViewController.swift

```diff
@@ -0,0 +1,1033 @@
+			videoContentView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+			videoContentView.leftAnchor.constraint(equalTo: view.leftAnchor),
+			videoContentView.rightAnchor.constraint(equalTo: view.rightAnchor),
+		])
+		NSLayoutConstraint.activate([
+			mainGestureView.topAnchor.constraint(equalTo: videoContentView.topAnchor),
+			mainGestureView.bottomAnchor.constraint(equalTo: videoContentView.bottomAnchor),
+			mainGestureView.leftAnchor.constraint(equalTo: videoContentView.leftAnchor),
+			mainGestureView.rightAnchor.constraint(equalTo: videoContentView.rightAnchor),
+		])
+		NSLayoutConstraint.activate([
+			systemControlOverlayLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+			systemControlOverlayLabel.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+		])
+	}
+
+	// MARK: viewWillDisappear
+
+	override func viewWillDisappear(_ animated: Bool) {
+		super.viewWillDisappear(animated)
+
+		NotificationCenter.default.removeObserver(self)
+	}
+
+	// MARK: viewDidLoad
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+
+		setupSubviews()
+		setupConstraints()
+
+		view.backgroundColor = .black
+		view.accessibilityIgnoresInvertColors = true
+
+		setupMediaPlayer(newViewModel: viewModel)
+
+		refreshJumpBackwardOverlayView(with: viewModel.jumpBackwardLength)
+		refreshJumpForwardOverlayView(with: viewModel.jumpForwardLength)
+
+		let defaultNotificationCenter = NotificationCenter.default
+		defaultNotificationCenter.addObserver(self, selector: #selector(appWillTerminate), name: UIApplication.willTerminateNotification,
+		                                      object: nil)
+		defaultNotificationCenter.addObserver(self, selector: #selector(appWillResignActive),
+		                                      name: UIApplication.willResignActiveNotification, object: nil)
+		defaultNotificationCenter.addObserver(self, selector: #selector(appWillResignActive),
+		                                      name: UIApplication.didEnterBackgroundNotification, object: nil)
+	}
+
+	@objc
+	private func appWillTerminate() {
+		viewModel.sendStopReport()
+	}
+
+	@objc
+	private func appWillResignActive() {
+		hideChaptersOverlay()
+
+		showOverlay()
+
+		stopOverlayDismissTimer()
+
+		vlcMediaPlayer.pause()
+
+		viewModel.sendPauseReport(paused: true)
+	}
+
+	override func viewDidAppear(_ animated: Bool) {
+		super.viewDidAppear(animated)
+
+		startPlayback()
+	}
+
+	override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+		if isScreenFilled {
+			fillScreen(screenSize: size)
+		}
+		super.viewWillTransition(to: size, with: coordinator)
+	}
+
+	// MARK: VideoContentView
+
+	private func makeVideoContentView() -> UIView {
+		let view = UIView()
+		view.translatesAutoresizingMaskIntoConstraints = false
+		view.backgroundColor = .black
+
+		return view
+	}
+
+	// MARK: MainGestureView
+
+	private func makeMainGestureView() -> UIView {
```

Comment:
With the gesture stuff being talked about in other PRs, you might have the responsibility of implementing whatever we decide in these tvOS views.

Reply:
So, unfortunately it seems that `ASCollectionView` does not support tvOS, and for now there are no plans to add support: https://github.com/apptekstudios/ASCollectionView/issues/138

I discovered this after migrating my iOS LiveTV implementation over to `ASCollectionView` when I went to move on to move tvOS over to it as well. This is unfortunate because the `ASCollectionView` api was easier/simpler than `SwiftUICollection` and just generally felt better to work with. 

Options for moving forward:
1. Stick with `SwiftUICollection` for LiveTV on both iOS/tvOS
2. Add `ASCollectionView` and use it for LiveTV on iOS, while leaving the tvOS implementation on `SwiftUICollection`
3. Some other solution 

`ASCollectionView` definitely feels better to work with, but using it on iOS would then also mean further divergence between the implementations on iOS vs tvOS. 

@LePips What would be your preferred solution here?